cbdefense_connector.py

```diff
@@ -713,7 +736,10 @@ def _handle_update_policy(self, param):
         try:
             data = json.loads(param["policy"])
         except Exception as e:
-            return action_result.set_status(phantom.APP_ERROR, "Policy needs to be valid JSON data: " + self._get_error_message_from_exception(e))
+            return action_result.set_status(
+                phantom.APP_ERROR,
+                "Policy needs to be valid JSON data: " + self._get_error_message_from_exception(e)
```

Comment:
Is there a reason why this is a concatenation rather than a string format? I know this is a next to main merge so it must not have been caught earlier. Not sure if it is worth changing at this point? If it is too much trouble feel free to ignore.

Reply:
Resolved